cartoonfight/player.py

```diff
@@ -25,13 +25,16 @@ def __init__(self, position, maxHealth, size, isPlayerOne, display):
 
         self.right = False
         self.left = False
+        self.jumping = False
 
+        self.base_atk_left = False
+        self.base_atk_right = False
```

Comment:
I think a better naming would be:
`basic_attack_left` and `basic_attack_right`
There is no need to abbreviation